changelogs/unreleased/4073-pradeep288

```diff
@@ -0,0 +1,39 @@
+Display which plugins are mandatory and can't be removed
```

Comment:
Could you please make the changelog more simplified?
Because the changelog will be shown in the GitHub release.